frontend/src/components/Board/OrderPCBs.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react'
+import Image from 'next/image'
 import { object, string } from 'prop-types'
 import { Icon } from 'semantic-ui-react'
 
 import styles from './OrderPCBs.module.scss'
 
+const retailerLogosDimensions = { height: 30, width: 100 }
+
```

Comment:
I believe you can get the dimensions automatically via `import`. 

https://nextjs.org/blog/next-11#automatic-size-detection-local-images

Reply:
They don't have the same size, I guess the point of using `Image` over `img` is the optimization(resize).